crates/pallet-executor/src/lib.rs

```diff
@@ -192,7 +211,12 @@ mod pallet {
                 fraud_proof
             );
 
-            // TODO: revert the execution chain.
+            // Revert the execution chain.
```

Comment:
It seems to me that pre-dispatch and validation logic will prevent this revert from actually going through. Am I missing something?

Reply:
I'm sorry, you're right, I didn't expand a piece of code to see that this is a new call.